src/Integration.Vsix.UnitTests/Settings/OtherOptionsDialogPageTests.cs

```diff
@@ -198,5 +197,25 @@ public void OnApply_WhenApplyBehaviorIsApplyAndShareDataBecomesFalse_OnlyCallsOp
             isOptInCalled.Should().BeFalse();
             isOptOutCalled.Should().BeTrue();
         }
+
+        private OtherOptionsDialogPageTestable CreateTestSubject(ITelemetryManager telemetryManager)
```

Comment:
Probably worth using the `[TestInitialize]` method instead of this.

Reply:
The individual tests configure the telemetry manager differently so this seemed neater.

Reply:
indeed